scripts/core/agent/agent.js

```diff
@@ -174,12 +202,17 @@ define([
             decideGoal();
             moveTo(newLocation);
             spendHunger(this.hungerRate);
+            spendEnergy(this.exhaustionRate);
+
+            this.age += 0.1;
```

Comment:
Do we need a "age" variable here that keeps increasing? I normally just have a "born" time and the age is simply now - born.